modules/openapi-generator/src/main/resources/go/api.mustache

```diff
@@ -3,6 +3,7 @@ package {{packageName}}
 
 {{#operations}}
 import (
+  "bytes"
```

Comment:
@code-lucidal58 what about using a tab instead?

Reply:
I use tabs. And looks fine in Atom and Intellij. Not sure why it looks different here. Let me try again

Reply:
done